staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_apis.go

```diff
@@ -81,11 +150,22 @@ func (r *apisHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	apiServicesByGroup := apiregistrationv1apihelper.SortedByGroupAndVersion(apiServices)
 	for _, apiGroupServers := range apiServicesByGroup {
-		// skip the legacy group
+		// special case the legacy group
+		var discoveryGroup *metav1.APIGroup
 		if len(apiGroupServers[0].Spec.Group) == 0 {
-			continue
+			groupVersionForDiscovery := metav1.GroupVersionForDiscovery{
```

Comment:
I'm guessing this is special handling for the legacy group? See my other comment nearby for why I don't think this is necessary.